### ISSUE TITLE
`bin/rails db:schema:load` broken

### ISSUE THREAD

**Issue by roschaefer** [ifad/chronomodel#55]
When I do:

```sh
$ bin/rails db:drop db:create db:schema:load RAILS_ENV=test
Dropped database 'rundfunk-backend-test'
Created database 'rundfunk-backend-test'
> psql -U -f /home/robert/Development/Rails/rundfunk-mitbestimmen/backend/db/structure.sql rundfunk-backend-test... exit 2
```

Shouldn't it be something like this?
```
psql  -d rundfunk-backend-test  -f db/structure.sql
```

It let's our build server fail which is doing `db:schema:load`.

**Comment by roschaefer** [comment]
On my laptop and our build server, we don't have an explicitly configured database user, so I guess at this line:
https://github.com/ifad/chronomodel/blob/ab3dbde7031fa7e135b4ad119966fbaa15bfc012/lib/chrono_model/schema_format.rb#L39
username is `nil` and the `-d` before database is missing too.

I can take care of this, I'll send a PR this weekend.

**Comment by vjt** [comment]
As reported, fixed by #51, needs review. Thanks.

**Comment by vjt** [comment]
Fixed by merging #51 in #60. Released as v0.12.0.
